packages/expo-modules-core/android/src/main/cpp/JavaCallback.h

```diff
@@ -6,23 +6,55 @@
 
 #include <fbjni/fbjni.h>
 #include <folly/dynamic.h>
+#include <variant>
 
 #include <react/jni/WritableNativeArray.h>
 #include <react/jni/WritableNativeMap.h>
+#include <fbjni/detail/CoreClasses.h>
 
 namespace jni = facebook::jni;
 namespace react = facebook::react;
 
 namespace expo {
+
+struct SharedObjectId : public jni::HybridClass<SharedObjectId> {
+  static constexpr const char *kJavaDescriptor = "Lexpo/modules/kotlin/sharedobjects/SharedObjectId;";
+
+  SharedObjectId();
+
+  RN_EXPORT int value{};
```

Comment:
```suggestion
  int value{};
```
Is that needed? If so, what's this doing?